### DISCUSSION TITLE
Column view when?

### DISCUSSION THREAD

**Discussion by onjour** [spacedriveapp/spacedrive#2413] [General]
I really appreciate what you do and what Spacedrive could offer.

For my use case, it's really hard to justify using it over other products like Path Finder given how the UI is. It's pretty, but why is everything so big? When dealing with thousands of files it's really inefficient. Plus there's no tree browsing yet.

I do have a lot of hopes for your upcoming "Column view", how are things looking? Can we expect it for 0.2.xx like stated on your roadmap?

Thanks and cheers

**Comment by utkubakir** [comment]
Column view is being worked on, should come out in the following month.

> It's pretty, but why is everything so big?

For most things, you can change their size from explorer settings on top right. If you mean something else or were looking for a even smaller size, let me know :)

**Comment by onjour** [comment]
> For most things, you can change their size from explorer settings on top right. If you mean something else or were looking for a even smaller size, let me know :)

Thank you for your reply, I am indeed looking for a smaller size. Compare a plain Finder window listing with one with Spacedrive _at it's smallest possible view_: it's still way bigger than MacOS — almost **twice**. Don't get me wrong @utkubakir, it's pretty and neat but it's very inefficient.

If you could make it at least equivalent that'd be great be I guess that won't happen until column view?

![Screenshot 2024-05-06 at 11 57 44](https://github.com/spacedriveapp/spacedrive/assets/39539070/a69c3794-acc9-4ea4-bff5-89a23fb33ee8)
